server/routes/activities/record-attendance/handlers/resetAttendance.ts

```diff
@@ -35,6 +36,11 @@ export default class ResetAttendanceRoutes {
     const { attendanceId } = req.params
     const { confirm } = req.body
 
+    const returnUrl =
```

Comment:
Need to redirect the user back to the original view based on whether they are attending a single activity instance or multiple ones